core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -394,6 +401,7 @@ class ContainerPool(
     private val warmNodejsKey = WarmNodeJsActionContainerId
     private val nodejsExec = NodeJS6Exec("", None)
     private val WARM_NODEJS_CONTAINERS = 2
+    private val RM_THRESHOLD = 4
```

Comment:
I shouldn't say it's entirely arbitrary but it's not dependent on the number of core but rather on the tolerable backlog of work based on the performance of `docker rm`.